internal/pkg/template/repository/manifest/records.go

```diff
@@ -103,7 +103,8 @@ func (v *TemplateRecord) DefaultVersion() (VersionRecord, bool) {
 	latest := VersionRecord{Version: model.ZeroSemVersion()}
 	latestStable := VersionRecord{Version: model.ZeroSemVersion()}
 	for _, item := range v.Versions {
-		if item.Version.GreaterThan(latest.Version.Value()) {
+		// GreaterThanOrEqual
+		if !item.Version.LessThan(latest.Version.Value()) {
```

Comment:
Monika pouziva v sablonach verziu `v0.0.1` a `ZeroSemVersion` vrati tiez `v0.0.1`, a teda podmienka je false.
Opravil som to.